subprojects/packagefiles/libupnp/meson_options.txt

```diff
@@ -0,0 +1,74 @@
+option('scriptsupport', type : 'boolean',
+  value : true,
+  description : 'script support for IXML document tree, see ixml.h',
+)
+
+option('samples', type : 'boolean',
+  value : true,
+  description : 'compilation of upnp/sample/ code',
+)
+
+option('blocking_tcp_connectionsr', type : 'boolean',
+  value : false,
+  description : 'blocking TCP connections',
+)
+
+option('client', type : 'boolean',
+  value : true,
+  description : 'control point code (client)',
+)
+
+option('device', type : 'boolean',
+  value : true,
+  description : 'device specific code (implies --disable-webserver if disabled)',
+)
+
+option('gena', type : 'boolean',
+  value : true,
+  description : 'GENA part',
+)
+
+option('tools', type : 'boolean',
+  value : true,
+  description : 'helper APIs in upnptools.h',
+)
+
+option('ipv6', type : 'boolean',
+  value : true,
+  description : 'ipv6 support',
+)
+
+option('optssdp', type : 'boolean',
+  value : true,
+  description : 'optional SSDP headers support',
+)
+
+option('open_ssl', type : 'boolean',
+  value : false,
+  description : 'open-ssl support',
```

Comment:
Huh, I am not used to seeing that spelled with a dash/underscore in the name. :p

Reply:
Upstream does this. No idea why.